lib/rules/at-rule-empty-line-before/__tests__/index.js

```diff
@@ -297,6 +297,26 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: ['always', { ignore: ['after-comment'] }],
+	syntax: 'scss',
+	fix: true,
+
+	accept: [
+		{
+			code: '// foo\n// bar\n@media {}',
+		},
+		{
+			code: '// foo\n// bar\n\n@media{}',
+		},
+		{
+			code: '// foo\r\n// bar\r\n\r\n@media {}',
+			description: 'CRLF',
+		},
+	],
+});
+
```

Comment:
@jeddy3 I’ve separated out the test cases as you suggested, but it still passes. Any ideas?

Reply:
Never mind. I’m closing this PR, as the issue seems to have been resolved.